utils/sf-config.js

```diff
@@ -7,57 +7,101 @@
 
 const { dirname } = require('path');
 const { cosmiconfigSync } = require('cosmiconfig');
-const { isMultiPackageProject } = require('./project-type');
+const { isPlugin } = require('./project-type');
 
 const PACKAGE_DEFAULTS = {
   scripts: {
-    build: 'sf-build',
+    build: 'wireit',
     clean: 'sf-clean',
     'clean-all': 'sf-clean all',
-    compile: 'sf-compile',
+    compile: 'wireit',
     docs: 'sf-docs',
-    format: 'sf-format',
-    // Cleaner errors than ts-node through tests
-    pretest: 'sf-compile-test',
-    test: 'sf-test',
-    lint: 'sf-lint',
+    format: 'wireit',
+    // this will be removed
+    pretest: undefined,
+    posttest: undefined,
+    test: 'wireit',
+    'test:compile': 'wireit',
+    'test:only': 'wireit',
+    lint: 'wireit',
     prepack: 'sf-prepack',
   },
-};
-
-const LERNA_DEFAULTS = {
-  scripts: {
-    build: 'lerna build',
-    clean: 'lerna clean',
-    'clean-all': 'lerna sf-clean all',
-    compile: 'lerna compile',
-    docs: 'lerna docs',
-    format: 'lerna format',
-    test: 'lerna test',
-    lint: 'lerna lint',
+  wireit: {
+    build: {
+      dependencies: ['compile', 'lint'],
+    },
+    compile: {
+      command: 'tsc -p . --pretty --incremental',
+      files: ['src/**/*.ts', 'tsconfig.json', 'messages/**'],
+      output: ['lib/**', '*.tsbuildinfo'],
+      clean: 'if-file-deleted',
+    },
+    format: {
+      command: 'prettier --write "+(src|test|schemas)/**/*.+(ts|js|json)|command-snapshot.json"',
```

Comment:
This could have `files` and `output` set to support caching

Reply:
and prettier config files 😄